packages/vow/src/E.js

```diff
@@ -327,7 +329,7 @@ export default makeE;
 /**
  * @template T
  * @typedef {{
- *   readonly [P in keyof T]: T[P] extends PromiseLike<infer U>
+ *   readonly [P in keyof T]: T[P] extends PromiseLike<infer _>
```

Comment:
since the binding isn't used,
```suggestion
 *   readonly [P in keyof T]: T[P] extends PromiseLike<any>
```

Reply:
Done, among other `infer _`s.